package.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>osqp</name>
+  <version>0.6.2</version>
+  <description>The Operator Splitting QP Solver</description>
+  <maintainer email="tbd@tbd.tbd">Bartolomeo Stellato</maintainer>
+
+  <license>>Apache License 2.0</license>
+
+  <build_depend>git</build_depend>
+
+  <buildtool_depend>cmake</buildtool_depend>
```

Comment:
Can you also add an exec_depend on catkin for ROS1 as in https://github.com/ori-drs/osqp/blob/f348102020f8a1c756ea6fe25553a38d01603059/package.xml#L14?

Reply:
Thank you!